Index spi (#162)
pinot-contrib/pinot-docs: basics/indexing/geospatial-support.md

{% code title="geoindex schema" %}
```javascript
{
  "dataType": "BYTES",
  "name": "location_st_point",
  "transformFunction": "toSphericalGeography(stPoint(lon,lat))"
}
```
{% endcode %}

Note the use of `transformFunction` that converts the created point into `SphericalGeography` format, which is needed by the `ST_Distance` function.

> Next, declare the geospatial index in the [table config](../../configuration-reference/table.md) you need to

Reviewer: ```suggestion
Next, declare the geospatial index in the [table config](../../configuration-reference/table.md) by doing the following:
```

> - Make sure the dictionary is disabled (see [dictionary documentation page](dictionary-index.md)).

Reviewer: ```suggestion
- Verify the dictionary is disabled (see how to [disable the dictionary index](dictionary-index.md)).
```

- Enable the H3 index.

> It is recommended to do the latter by using the `indexes` section:

Reviewer: ```suggestion
We recommend doing the latter by using the `indexes` section:
```

Reviewer: doing the latter--does this mean enabling the H3 index?

{% code title="geoindex tableConfig" %}
```javascript
{
  "fieldConfigList": [
    {
      "name": "location_st_point",
      "encodingType":"RAW", // this actually disables the dictionary
      "indexes": {
        "h3": {

>           "resolutions": [13, 5, 6]

Author: I've changed the example to use a resolution with multiple entries in order to see the differences between the new and the old syntax
